lib/utils/utils.js

```diff
@@ -213,12 +213,22 @@ function getSourceModuleNameForIdentifier(context, node) {
   const importDeclaration = program.body
     .filter(isImportDeclaration)
     .find(importDeclaration =>
-      importDeclaration.specifiers.some(specifier => specifier.local.name === node.name)
+      importDeclaration.specifiers.some(
+        specifier => specifier.local.name === getSourceModuleName(node)
+      )
     );
 
   return importDeclaration ? importDeclaration.source.value : undefined;
 }
 
+function getSourceModuleName(node) {
```

Comment:
Can you add unit tests for `getSourceModuleName` and `getSourceModuleNameForIdentifier` to ensure the new functionality is correct?